lib/aur-build

```diff
@@ -59,93 +56,95 @@ var_tmp=$(mktemp -d "${TMPDIR:-/var/tmp}/$argv0".XXXXXXXX)
 trap 'trap_exit' EXIT
 trap 'exit' INT
 
-opt_short='a:d:r:C:D:M:cfNRsv'
-opt_long=()
+## option parsing
+opt_short='a:d:p:r:D:cfNRsv'
+opt_long=('arg-file:' 'chroot' 'database:' 'force' 'path:' 'root:'
+          'sign' 'verify' 'directory:' 'no-sync' 'pacman-conf:'
+          'makepkg-conf:' 'remove')
+
 if ! parseopts "$opt_short" "${opt_long[@]}" -- "$@"; then
     usage
 fi
 set -- "${OPTRET[@]}"
 
-unset db_name db_root queue
+unset queue db_name db_path db_root
 while true; do
     case "$1" in
-        -a) shift; queue=$1 ;;
-        -c) chroot=1 ;;
-        -f) overwrite=1 ;;
-        -N) no_sync=1 ;;
-        -r) shift; db_root=$1 ;;
-        -d) shift; db_name=$1
-            chroot_args+=(-d "$1") ;;
-        -C) shift; chroot_args+=(-C "$1") ;;
-        -D) shift; chroot_args+=(-D "$1") ;;
-        -M) shift; chroot_args+=(-M "$1") ;;
-        -R) repo_add_args+=(-R) ;;
-        -s) sign_pkg=1
-            repo_add_args+=(-s) ;;
-        -v) repo_add_args+=(-v) ;;
+        -a|--arg-file)  shift; queue=$1 ;;
+        -p|--path)      shift; db_path=$1 ;;
+        -d|--database)  shift; db_name=$1
+                        repo_args+=(-d "$1") ;;
+        -r|--root)      shift;
+                        repo_args+=(-r "$1") ;;
+        -c|--chroot)    chroot=1 ;;
+        -f|--force)     overwrite=1 ;;
+        -s|--sign)      sign_pkg=1
+                        repo_add_args+=(-s) ;;
+        -v|--verify)    repo_add_args+=(-v) ;;
+        -D|--directory) shift; chroot_args+=(-D "$1") ;;
+        -N|--no-sync)   no_sync=1 ;;
+        -R|--remove)    repo_add_args+=(-R) ;;
+        --pacman-conf)  shift; chroot_args+=(-C "$1") ;;
+        --makepkg-conf) shift; chroot_args+=(-M "$1") ;;
```

Comment:
Nitpick: this would have been better to make these changes in commit 90a317513a311bb430983606f73597aa3b2dfa2b, (add long options) since this is independent of "aur-repo: implement --status-file (#448)"

Reply:
More fun with `git rebase`...